Tasks/RunLoadTest/task.json

```diff
@@ -0,0 +1,89 @@
+﻿{
+  "id": "9E9DB38A-B40B-4C13-B7F0-31031C894C22",
+  "name": "CloudLoadTest",
+  "friendlyName": "Cloud-based Load Test with VSO",
+  "description": "Runs the load test in cloud; Powered by VS Online.",
+  "category": "Test",
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 1,
+    "Minor": 0,
+    "Patch": 1
+  },
+  "demands": [
+    "msbuild"
+  ],
+  "groups": [
```

Comment:
Doesn't this need a visibility or Preview?  We can't release tasks that take tfs urls and first party creds